flink-streaming-java/src/main/java/org/apache/flink/streaming/api/RuntimeExecutionMode.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.api;
+
+import org.apache.flink.annotation.Internal;
+
+/**
+ * Runtime execution mode of DataStream programs. Among other things, this controls task scheduling,
+ * network shuffle behavior, and time semantics. Some operations will also change their record emission behaviour
+ * based on the configured execution mode.
+ *
+ * @see <a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-134%3A+Batch+execution+for+the+DataStream+API">
+ *     https://cwiki.apache.org/confluence/display/FLINK/FLIP-134%3A+Batch+execution+for+the+DataStream+API</a>
+ */
+@Internal
+public enum RuntimeExecutionMode {
+
+	/**
+	 * The Pipeline will be executed with Streaming Semantics. All tasks will be deployed before execution starts,
+	 * checkpoints will be enabled, and both processing and event time will be fully supported.
+	 *
+	 * @see <a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-134%3A+Batch+execution+for+the+DataStream+API">
```

Comment:
We probably don't need the `@see` link for all variants. 😅

Reply:
I thought I had removed them already.